electron/main.js

```diff
@@ -169,13 +169,17 @@ function showMainWindow() {
     main.setBounds(init.restore('bounds', main.getBounds()));
   }
 
-  main.webContents.session.setCertificateVerifyProc((hostname = '', certificate = {}, cb) => {
-    const {data: cert = ''} = certificate;
+  main.webContents.session.setCertificateVerifyProc((request, cb) => {
+    const { hostname = '', certificate: { data: cert = '' } = {}, error } = request;
+
+    if (error !== undefined) {
+      return cb(0);
```

Comment:
Good idea - in this case then we don't need to double-check because the certificate won't be valid anyway.